sdk/storage/inc/blobs/blob_options.hpp

```diff
@@ -11,6 +12,65 @@
 
 namespace Azure { namespace Storage { namespace Blobs {
 
+  /**
+   * @brief Specifies access conditions for a container.
+   */
+  struct ContainerAccessConditions : public ModifiedTimeAccessConditions,
```

Comment:
How about `LastModifiedAccessCondition`, to align with returned value from response header.

Reply:
fixed